src/v0/destinations/google_adwords_remarketing_lists/networkHandler.js

```diff
@@ -99,7 +99,7 @@ const gaAudienceProxyRequest = async (request) => {
   // step1: offlineUserDataJobs creation
 
   const firstResponse = await createJob(endpoint, customerId, listId, headers, method);
-  if (!firstResponse.success && !isHttpStatusSuccess(firstResponse?.response?.response?.status)) {
+  if (!firstResponse.success && !isHttpStatusSuccess(firstResponse?.response?.status)) {
```

Comment:
Don't we need to check the partialFailureError scenario here?

Reply:
I haven't encountered any partial failure in case of firstResponse, but added as safety check